src/content/1.7/webservice/resources/content_management_system.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Content management system
+---
+
+# Resources for Content management system
+
+### Content
+
+|         Name         |    Format     | Required | Read Only |   Max size    | Not filterable | Description |
+| :------------------- | :------------ | :------- | :-------- | :------------ | :------------- | :---------- |
+| **id_cms_category**  | isUnsignedInt |          |           |               |                |             |
+| **position**         |               |          |           |               |                |             |
+| **indexation**       |               |          |           |               |                |             |
+| **active**           |               |          |           |               |                |             |
+| **meta_description** | isGenericName |          |           | 512           |                |             |
+| **meta_keywords**    | isGenericName |          |           | 255           |                |             |
+| **meta_title**       | isGenericName | true     |           | 255           |                |             |
+| **head_seo_title**   | isGenericName |          |           | 255           |                |             |
+| **link_rewrite**     | isLinkRewrite | true     |           | 128           |                |             |
+| **content**          | isCleanHtml   |          |           | 3999999999999 |                |             |
```

Comment:
```suggestion
| **content**          | isCleanHtml   |          |           | 3999999999999 |                |             |
```

3999999999999 : Really ?

Reply:
It's the value in the webservice API and in the code :sweat_smile:

Reply:
```
'content' => ['type' => self::TYPE_HTML, 'lang' => true, 'validate' => 'isCleanHtml', 'size' => 3999999999999],
```